source/agora/consensus/Ledger.d

```diff
@@ -1791,45 +1704,6 @@ unittest
     scope ledger = new TestLedger(WK.Keys.A, [GenesisBlock], params);
 }
 
-unittest
```

Comment:
Why is this test removed ?

Reply:
It was testing that when previous block has less than majority the next block is not externalized. Maybe I should bring it back but to test the current block is not externalized with less than majority.

Reply:
Added